doc/api/prev_api_changes/api_changes_3.0.0.rst

```diff
@@ -116,14 +116,14 @@ Different exception types for undocumented options
 Improved call signature for ``Axes.margins``
 --------------------------------------------
 
-:meth:`matplotlib.axes.Axes.margins` and :meth:`mpl_toolkits.mplot3d.Axes3D.margins`
+`.Axes.margins` and `.Axes3D.margins`
```

Comment:
Can we now use `.Axes` without module? I thought there was a problem with ambigity of `axes.Axes` and `axes_grid1.mpl_axes.Axes`?

Reply:
For (most) *method* names it's fine because `matplotlib.axes.Axes.margins` has a doc entry, but `mpl_toolkits.axes_grid1.mpl_axes.Axes.margins` doesn't.

Reply:
Ah, so it does not depend on the existence of the second method but on the existence of the second docstring.